src/averageAges.js

```diff
@@ -15,11 +15,20 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
   // learn how to use array methods like .filter .map .some .every .find .reduce
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const onlyMen = century
+    ? people.filter(men => men.sex === 'm'
+      && Math.ceil(men.died / 100) === century)
+    : people.filter(men => men.sex === 'm');
+
+  const sumOfAge = onlyMen.reduce((sum, men) => sum + (men.died - men.born), 0);
+
+  const menAverageAge = sumOfAge / onlyMen.length;
+
```

Comment:
![image](https://user-images.githubusercontent.com/59209337/207680641-46b6cfdf-111e-49ee-85f0-d181908c9aa2.png)
you didn't fix it